app/controllers/admin/enterprises_controller.rb

```diff
@@ -22,6 +24,9 @@ class EnterprisesController < Admin::ResourceController
     before_action :load_properties, only: [:edit, :update]
     before_action :setup_property, only: [:edit]
 
+    update.after  :geocode_address_if_use_geocoder
+    create.after  :geocode_address_if_use_geocoder
```

Comment:
I just reviewed #7174 which will remove this type of callback. We need to replace it with Rails' callbacks.

```suggestion
    after_action :geocode_address_if_use_geocoder, only: :update
    after_action :geocode_address_if_use_geocoder, only: :create
```

Is this right, @Matt-Yorkley?

Reply:
It can be one line, right?
```ruby
after_action :geocode_address_if_use_geocoder, only: [:create, :update]
```

Reply:
Ha! Yeah, my brain worked as stupidly as a machine. :smile: